api/garden.yml

```diff
@@ -25,7 +25,7 @@ spec:
   ingresses:
     - path: /
       port: http
-      hostname: api.${variables.base-hostname}
+      hostname: api-${variables.base-hostname}
```

Comment:
I've changed the URLs to avoid SSL errors when using this example with the AWS quickstart cluster: https://github.com/garden-io/garden-aws-quickstart

Reply:
Did we also update DNS?

Reply:
I need to test the local version of this 👍